docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -487,22 +496,67 @@ export default function AppLayoutDocsFooter(props) {
                   {t('submit')}
                 </Button>
               </DialogActions>
-              {rating !== 1 && (
-                <Typography id="feedback-description" color="text.secondary">
-                  {t('feedbackMessageToGitHub.usecases')}
-                  <br />
-                  {t('feedbackMessageToGitHub.callToAction.text')}
+            </Box>
+          </form>
+        </Collapse>
+        <Divider sx={{ mb: 4 }} />
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: { xs: 'column', md: 'row' },
+            justifyContent: 'space-between',
+            gap: { xs: 4, lg: 12 },
+          }}
+        >
+          <Box flexGrow={1}>
+            <Typography variant="body2" fontWeight="bold">
+              Keep up to date
+            </Typography>
+            <Typography variant="body2" color="text.secondary" sx={{ mb: 2 }}>
+              Join our newsletter for regular updates. No spam ever.
+            </Typography>
+            <EmailSubscribe fullWidth sx={{ mb: 1 }} />
+          </Box>
+          <Divider sx={{ display: { xs: 'block', sm: 'none' } }} />
+          <Stack direction="column" spacing={3}>
+            <Typography variant="body2" fontWeight="bold">
+              Engage further with us
+            </Typography>
+            <Stack direction="column" spacing={1}>
+              <Stack direction="row" alignItems="center" spacing={1}>
+                <BugReportRoundedIcon fontSize="small" color="disabled" />
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 0 }}>
+                  Need answers?
                   <Link
                     href={`${process.env.SOURCE_CODE_REPO}/issues/new?template=${process.env.GITHUB_TEMPLATE_DOCS_FEEDBACK}&page-url=${window.location.href}`}
                     target="_blank"
+                    sx={{ ml: 0.5 }}
                   >
                     {t('feedbackMessageToGitHub.callToAction.link')}
                   </Link>
                 </Typography>
-              )}
-            </Box>
-          </form>
-        </Collapse>
+              </Stack>
+              <Stack direction="row" alignItems="center" spacing={1}>
+                <ChangeCircleRoundedIcon fontSize="small" color="disabled" />
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 0 }}>
+                  Past releases?
+                  <Link href={ROUTES.roadmap} target="_blank" sx={{ ml: 0.5 }}>
+                    Check the changelog
+                  </Link>
+                </Typography>
+              </Stack>
+              <Stack direction="row" alignItems="center" spacing={1}>
+                <MapRoundedIcon fontSize="small" color="disabled" />
+                <Typography variant="body2" color="text.secondary" sx={{ mb: 0 }}>
+                  The future?
+                  <Link href={ROUTES.roadmap} target="_blank" sx={{ ml: 0.5 }}>
```

Comment:
What problem does the Routes abstraction solve that inlinig the value doesn't?

Reply:
Also, how do we make it work with MUI X and Toolpad docs?

Reply:
I'm not 100% sure what you mean with the first question but for the second, and related to the comment above, the links are not right as ideally, they'd be dynamic as per docs ⎯ in other words, if I'm within the MUI X docs, the "roadmap" link takes me to the X link; if I'm within the Material UI docs, the "roadmap" link takes me to the Core link; etc. Definitely don't know how to pull this off, though 😅

Reply:
> I'm not 100% sure what you mean with the first question but for the second

I feel like we could inline the value directly, it seem simpler.

> Definitely don't know how to pull this off, though

I think that the best setup is for each docs to add the link to the roadmap as an env variable.

Reply:
@siriwatknp mind giving me tips/instructions on how to get to the result @oliviertassinari suggested for the links?